.github/workflows/apply-knowledge.yaml

```diff
@@ -46,7 +46,7 @@ jobs:
         uses: JoseThen/comment-pr@v1.2.0
```

Comment:
Replace this whole step by inlining our GitHub comment helper into the apply knowledge script